packages/next-server/lib/head.js

```diff
@@ -108,4 +107,6 @@ if (process.env.NODE_ENV === 'development') {
   })
 }
 
+export { HeadManagerContext }
```

Comment:
Why is this export needed?

Reply:
It’s not really, I just assumed we wanted to expose the context but now I don’t see why

Reply:
Let's drop it, headmanager is not a public API 🙏